src/components/GradeLevelFilter.jsx

```diff
@@ -56,24 +54,23 @@ export default function GradeLevelFilter() {
         Grade
       </Button>
       <Popover
-        className={classes.popOverWindow}
         open={Boolean(anchorEl)}
         anchorEl={anchorEl}
         onClose={() => setAnchorEl(null)}
         anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
         transformOrigin={{ vertical: 'top', horizontal: 'left' }}>
-        {Object.keys(gradeList).map((key) => (
-          <>
+        {Object.keys(gradeItems).map((key) => (
```

Comment:
Hmm, I'm not entirely sure `Object.keys()` works with sets. It doesn't seem to work when I type this in the Web inspector JavaScript console:

```js
s = new Set(["bar", "foo"]);
Object.keys(s);  // outputs []
```

Another way you could write this is:
```suggestion
        {Array.from(gradeItems).map((grade) => (
```
And use `grade` for the key/value

Reply:
Hmm ok yea I see, made the changes